articles/active-directory/governance/entitlement-management-external-users.md

```diff
@@ -86,6 +86,9 @@ To ensure people outside of your organization can request access packages and ge
 - For more information about Azure AD B2B external collaboration settings, see [Configure external collaboration settings](../external-identities/external-collaboration-settings-configure.md).
 
     ![Azure AD external collaboration settings](./media/entitlement-management-external-users/collaboration-settings.png)
+    
+    > [!NOTE]
+    > If you create a connected organization for a cross-cloud Azure AD tenant, you also need to configure cross-tenant access settings appropriately. For more information on how to configure these settings, see [Configure cross-tenant access settings](../external-identities/cross-tenant-access-settings-b2b-collaboration.md).
```

Comment:
we should use this docs link: https://docs.microsoft.com/azure/active-directory/external-identities/cross-cloud-settings

Reply:
It will be live tomorrow morning at 10am PDT.

Reply:
Let's also use this verbiage (per marketing): If you create a connected organization for an Azure AD tenant from a different Microsoft cloud,...